### ISSUE TITLE
Block media upload if plan limit is met

### ISSUE THREAD

**Issue by FransdeJong** [umbraco/Umbraco.Cloud.Issues#562]
### Issue description

Since the automatic plan upgrade functionality is being deployed on Umbraco Cloud we would like to see a safeguard in place.

I would like a setting that prevents Umbraco from upgrading media when it will go over the plan limit. This setting can be optional/disabled by choice if a client chooses to do so or if a project isn't hosted on umbraco cloud since I think this would mean adding this functionality to Umbraco Cms.

**Comment by sajumb** [comment]
Thanks for the feature request. We have added it to the product backlog.

**Comment by FransdeJong** [comment]
Is this an issue for Cloud? Or should it be cross posted in the CMS?

I can imagine this being a great feature for people running a VPS as well?

I've added this issue more like a request/discussion.

**Comment by sajumb** [comment]
Yes, I fully agree. I will have a talk with our CMS Core team about this and see where we believe it would be best to add a solution. We will keep you in the loop.

**Comment by jemayn** [comment]
Thank you for reporting this. We're closing this issue as part of a cleanup of older items on the tracker. If you're still experiencing this problem on a current version of Umbraco Cloud, please open a new issue and we'll investigate.
